uk.co.compucorp.civicrm.tasksassignments/views/dashboard/documents.html

```diff
@@ -40,10 +40,14 @@ <h3 class="panel-title">{{label[filterParams.due]}}</h3>
                 ng-model="filterParams.ownership"
                 uib-btn-radio="'assigned'">
                 My Docs
-              <span>({{(list |
-                filterByDue:filterParams.due:filterParams.dateRange |
-                filterByStatus:filterParams.documentStatus |
-                filterByOwnership:'assigned').length}})
+              <span>
+                ({{
+                  ((((list | filterByDueExpiry:filterParams.due:'activity_date_time':filterParams.dateRange)).length ?
```

Comment:
Having this big conditions in the HTMl is hard to follow, its better to create scope function and use the filters in the JS itself.

Reply:
Moved to a function in controller.  Done!